docs/src/lib/remote-links.md

```diff
@@ -45,6 +44,18 @@ This is done regardless whether any additional remotes have been specified with
 
 You can think of it as Documenter automatically populating `remotes` with any cloned GitHub repositories it finds.[^3]
 
+For released packages (those added using `Pkg.add(...)` rather than `Pkg.develop(...)`), the version and repository can be determined from the package metadata, but a commit hash is not readily available. In this case, Documenter will guess that a tag `v$VERSION` exists in the repository on GitHub. Note that these tags are created automatically by the widely used JuliaRegistries/TagBot action. Since this is sometimes not the case, and could cause dead or incorrect links, the `GitHubCheckExt` extension module is available for Julia 1.9 once the `GitHub` module is installed. You can use it to check the links as part of your documentation build. Note that this will make GitHub API requests and so could cause builds to fail due to network errors or GitHub downtime. Simply run `import Pkg; Pkg.add("GitHub")` and then modify `build.jl` to look like the following:
```

Comment:
For cleaner diffs, I have started following a one line = one sentence structure in Markdown docs:

```suggestion
For released packages (those added using `Pkg.add(...)` rather than `Pkg.develop(...)`), the version and repository can be determined from the package metadata, but a commit hash is not readily available.
In this case, Documenter will guess that a tag `v$VERSION` exists in the repository on GitHub.
Note that these tags are created automatically by the widely used JuliaRegistries/TagBot action.
Since this is sometimes not the case, and could cause dead or incorrect links, the `GitHubCheckExt` extension module is available for Julia 1.9 once the `GitHub` module is installed.
You can use it to check the links as part of your documentation build.
Note that this will make GitHub API requests and so could cause builds to fail due to network errors or GitHub downtime.
Simply run `import Pkg; Pkg.add("GitHub")` and then modify `build.jl` to look like the following:
```